hypermap/aggregator/utils.py

```diff
@@ -331,7 +331,7 @@ def get_single_service(esri, url, endpoint):
     len_list = len(url_split_list)
 
     service_name = url_split_list[0]
-    if len_list == 3:
+    if len_list <= 3:
```

Comment:
Hey Jorge, as we discussed, let's find a more general way to solve this issue - but not hardcoding the number of subfolders.